extras/uwsgi/uwsgi.service

```diff
@@ -1,13 +0,0 @@
-[Unit]
-Description=uWSGI Emperor
-
-[Service]
-ExecStart=/usr/bin/uwsgi --emperor /etc/uwsgi/apps-enabled --logto /var/log/uwsgi/access.log
-RuntimeDirectory=uwsgi
-Restart=always
-KillSignal=SIGQUIT
-Type=notify
-NotifyAccess=all
-
-[Install]
-WantedBy=multi-user.target
```

Comment:
Do note that you don't need the whole `uwsgi.service` unit file.

You only need to add a `/etc/systemd/system/uwsgi.service.d/after-mongodb.conf` file, containing:

```ini
[Unit]
After=mongodb.service
```

And that's it!